src/styles/card.scss

```
@@ -0,0 +1,95 @@
+.card {
+  width: $card-width;
+  height: 408px;
+  border-radius: 5px;
+  border: 1px solid $border-color;
+  padding: 32px 16px 0;
+  transition: all 300ms;
+
+  &:hover {
+    transform: scale(120%);
+  }
+
+  &__image {
+    display: flex;
+    margin: auto;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    display: block;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $main-accent;
+    margin: 40px 0 4px;
```

reset margin for all `p`, `h2` to avoid reseting at each selector
```suggestion
    margin-top: 40px;
```